sickbeard/manual_snatch.py

```diff
@@ -170,9 +171,11 @@ def get_provider_cache_results(indexer, show_all_results=None, perform_search=No
     sql_return = []
     found_items = []
     provider_results = {'last_prov_updates': {}, 'error': {}, 'found_items': []}
+    original_thread_name = threading.currentThread().name
 
-    providers = [x for x in sickbeard.providers.sortedProviderList(sickbeard.RANDOMIZE_PROVIDERS) if x.is_active() and x.enable_daily]
+    providers = [x for x in sickbeard.providers.sortedProviderList(sickbeard.RANDOMIZE_PROVIDERS) if x.is_active() and x.enable_manualsearch]
     for curProvider in providers:
+        threading.currentThread().name = original_thread_name + " :: [" + curProvider.name + "]"
```

Comment:
Except if as you stated [here](https://github.com/pymedusa/SickRage/pull/207#issuecomment-201911631) "don't have time now to go over 100 files. Sorry. wont do it now. maybe when manual search and everything else is done".  Doing this when you are working on a section breaks this up into smaller jobs so it all doesn't have to be done at once.  Also as there is no consistency currently, and this is the consistent standard we are going for, there is no reason that this shouldnt be done.  Not at the start of the PR, but when getting it ready for merge.